driver-nats/src/main/java/io/openmessaging/benchmark/driver/nats/NatsBenchmarkDriver.java

```diff
@@ -76,12 +76,12 @@
         Connection cn;
         log.info("createConsumer");
         try {
-            Options options = new Options.Builder().server(config.natsHostUrl).maxReconnects(-1).build();
+            Options options = new Options.Builder().server(config.natsHostUrl).maxReconnects(5).build();
             cn = Nats.connect(options);
             natsConsumer = cn.createDispatcher((msg) -> {
                 consumerCallback.messageReceived(msg.getData(), Long.parseLong(msg.getReplyTo()));
             });
-            natsConsumer.subscribe(topic);
+            natsConsumer.subscribe(topic, subscriptionName);
```

Comment:
Great to see this being implemented!

This will create a queue subscriber, described on the bottom of the page in the [Queue Subscribers & Sharing Work](https://nats.io/documentation/writing_applications/concepts/) section.  From the name of the variable it seemed like you may intend it to be a label for the subscriber, which wouldn't be the case using this API - in the API that variable describes the queue group to join.  Is that the intent?